fasthttpadaptor/request.go

```diff
@@ -2,31 +2,31 @@ package fasthttpadaptor
 
 import (
 	"bytes"
+	"github.com/valyala/fasthttp"
 	"io/ioutil"
 	"net/http"
 	"net/url"
-
-	"github.com/valyala/fasthttp"
+	"unsafe"
```

Comment:
Moving the github import to a separate block is good. But can you move the `unsafe` import together with the block of other std imports?

Reply:
Thank you for review, I made the changes 👍